### PR TITLE
Breathing Oxygen Deep-crits Vox from Full-HP ~1min 30sec Faster

### PR DESCRIPTION
**Pull request by KasparoVy**
### This is a PR with two options.
**ONE:** PR stays as is: 25% increase to deadliness of breathing Oxygen (comparing pre-nerf state and Plasma-level deadliness).
**TWO:** PR changed so Oxygen will kill Vox just as quick as Plasma.

_Notice how I got that second commit in there? If it's determined this is to be taken all the way, I'll just peel a layer off the proverbial onion._

------------------------------------------------

Does what it says on the tin.

Oxygen is now right between being just as deadly as plasma to Vox and its current state in the live environment.

The alternative is to have max_oxy set to 0.005, which makes Vox drop into deep crit from full HP at the same time plasma would make them.

The figures:
------------------------------------------------
_Preface: I chose 'time to deep crit' because you stop taking breath-tox damage at that stage as you're technically no longer breathing._

max_oxy = time to deep crit
1	= ~05min 25sec -> Current value
0.75	= ~03min 55sec -> Middle of the road
0.5	= ~02min 25sec -> From here on, all max_oxy values kill you just about as quick.
			Might as well make it the same as plasma: 0.005.
0.05	= ~02min 15sec
0.005	= ~02min 10sec -> Just as deadly as plasma

------------------------------------------------

:cl:
tweak: The Vox now have a more severe aversion to Oxygen. Not breathing pure Oxygen now puts them in a deep-critical state ~1min 30sec/25% faster (takes ~3min 55sec vs. ~5min 25sec).
/:cl:

### CODE REVIEW

**Comment by Xantholne** [comment]
Inb4 voxxies start screaming it's too much and something about voxygen

**Comment by KasparoVy** [comment]
I included the figures for that reason, and I play the species regularly as good measure.

File: code/modules/mob/living/carbon/human/species/station.dm [hunk 1]
```diff
@@ -454,6 +454,19 @@
 	tail = "armalis_tail"
 	icon_template = 'icons/mob/human_races/r_armalis.dmi'
 
+	atmos_requirements = list(
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/species/station.dm]: Please do not change the armalis, they are already buggy, inconsistent, can't wear most masks, and barely function.

Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/species/station.dm]: This is just ensuring the Armalis aren't affected by the increased deadliness of Oxygen to their lesser counterparts the Primalis.

Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/species/station.dm]: Actually, you're right, I didn't notice they were actually a subtype of vox. Nevermind.

Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/species/station.dm]: All good, glad we could sort this out

**Comment by AffectedArc07** [comment]
:thumbsdown:

**Comment by SheepFighter** [comment]
As a vox main and veritable skrekbirb enthusiast, I've been waiting for something like this for a while. It's sorely needed. As it stands, their reliance on voxygen is barely noticeable. It actually amounts, as has been said many times before, to something of a benefit. 

So by all means make oxygen every bit as deadly as plasma. *Pleasies.*

**Comment by Spacemanspark** [comment]
@Affectedarc07 please provide a reasoning to your post, and not just a thumbsup/ thumbsdown reaction.

**Comment by AffectedArc07** [comment]
The reason for my thumbs down is because of this and the mouth covering blocking food PR. It this gets merged and the mouth covering gets merged, vox are going to need to make going to medbay a part of eating.

**Comment by SheepFighter** [comment]
@AffectedArc07 That's less to do with this PR and more with the existence of Vox, surely? It makes eating for them an undue pain either way.

**Comment by SamHPurp** [comment]
@AffectedArc07 Firstly that is largely an arguement for the OTHER PR, not this one. Secondly, that PR was closed.

**Comment by AffectedArc07** [comment]
Oh. In that case no objections.

**Comment by KasparoVy** [comment]
@SamHPurp I'd rather let the higher powers decide on the numbers, I'm not opposed to either as well. I'll think more on it when I got some time, but I feel if I just started this PR as making Oxygen as toxic as Plasma to Vox it would've been flamed

**Comment by FlattestGuitar** [comment]
To be fair, I don't really like this.

Ripping the mask off shouldn't insta-kill a Vox, but provide *massive stress* on them to find a mask and tank soon. 

I think with the amount being lowered too much, we might see more or less ALL Vox dying if they happen to lose their tank, which is just not interesting design-wise. I'm not sure what number would be adequate, however.

**Comment by Spartan6** [comment]
Vox already take the largest damage, do they need another damage boost to them?

**Comment by scrubmcnoob** [comment]
However,@FlattestGuitar 
Ripping the suit/helmet off a plasma man will set them on fire. Nothing nearly as drastic occurs to the Vox when their mask comes off.

**Comment by FlattestGuitar** [comment]
@scrubmcnoob, absolutely, but there's no real way to set people on fire without adding that sort of a sense of urgency, while here we can choose what we want.

**Comment by SirBloodBuske** [comment]
I play vox only, I like this

**Comment by KasparoVy** [comment]
The question you seem to have is "how long is long enough without being too quick?" and I can't answer that with any authority. I'll defer to the higher-ups on a decision like that.

**Comment by Allfd** [comment]
👍 

The less time this takes the better.

**Comment by TullyBurnalot** [comment]
Following discussion, it was decided to close this PR; lack of Voxygen is already deadly enough as it is. Making it MORE deadly, taking into consideration the amount of legitimate accidents that happen with Vox tanks, would be a bit overkill.

**Comment by KasparoVy** [comment]
Interesting decision. I can wait.

**Comment by Ralta** [comment]
Aw, closed already? As a regular Vox player, this would've been nice. It feels like taking your mask off or losing your voxxygen is an inconvenience more than a life-threatening emergency (unless it's removed maliciously in which case not having voxxygen is the least of your worries).
I've gone SSD at arrivals before and gotten up with my voxxy tank on the floor with a little toxin damage but not enough to seriously make me worry. 
Oh well. Hope this, or something like it, gets re-opened in the near future.

**Comment by SamHPurp** [comment]
Five minutes to enter crit because of lack of Voxygen is super hugboxy. I still support this merge....

**Comment by Allfd** [comment]
I feel like this is a continuation of the trend of making species human+ and I am sad to see that trend continue.

**Comment by taukausanake** [comment]
I'm just trying to figure out how deadly oxygen is to Vox. They breath nitrogen, N2, and breathable air is about 80% that stuff. Is the 20% really that dangerous to them?

**Pull request closed without merging**